docs/devctr-image.md

```diff
@@ -29,6 +31,14 @@ registry. The Firecracker CI suite must also be updated to use the new image.
    | docker login --username AWS --password-stdin public.ecr.aws
     ```
 
+   Although not recommended, an alternative approach that works with non-TTY
```

Comment:
Can mention that it's a less secure approach.

Reply:
Changed to:
> For non-TTY devices, although not recommended a less secure approach can be used`